sys/kern/syscalls.conf

```diff
@@ -1,6 +1,6 @@
 libsysmap="../../lib/libsys/syscalls.map"
 sysmk="../sys/syscall.mk"
-ptr_qualified="* __capability "
+ptr_qualified="* __kerncap "
```

Comment:
To be clear, is this so you can use the same _libsys.h for lib64 and lib64c?  How does that work currently for lib32 in stock FreeBSD?  Is it a shared header rather than per-ABI?

Reply:
Well `T *` is the native thing for lib{32,64,64c}, but using `__capability` makes that no longer true for lib64, hence `__kerncap` instead to only mess with things in hybrid kernel land.

Reply:
This should be a shared header that is correct for all ABIs (subject to cdefs.h being right.) Currently it's only used by libc and libthr, but hopefully we'll get language runtimes using it eventually.